### DISCUSSION TITLE
Quickest Way to Archive Multiple Quests

### DISCUSSION THREAD

**Discussion by fjbagley** [bytedeck/bytedeck#1382] [Q&A] [answered]
What is the best method to archive multiple quests?

**Comment by tylerecouture** [comment, marked as answer]
Via Django Admin (via Admin > Django Admin > Quests > Quests):

![image](https://github.com/bytedeck/bytedeck/assets/10604391/12d4bd18-9211-4c5f-93e9-24becac00094)
